### ISSUE TITLE
Simulating susceptibility-induced distortion fields.

### ISSUE THREAD

**Issue by alegouhy** [CMIC-UCL/CMICHACKS#10]
# Simulation of susceptibility-induced distortion fields.
Project leader: Antoine Legouhy [a.legouhy@ucl.ac.uk](mailto:a.legouhy@ucl.ac.uk)

# Description: ​
Trying to create a model to simulate susceptibility-induced distortion fields from whole-head segmentations.

##  Participant requirements: ​
Ideally some experience in at least one of the following:
python, tensorflow or pytorch, itk, image processing softwares.  

## Resources: ​
For about 60 subjects:
  1. Whole head segmentation maps obtained with Simnibs Charm from T1 and T2 images.
  2. Distortion fields obtained with FSL TOPUP from echo-planar images pairs with opposite PED.
 
Intra-subject registrations have been performed to align EPI and anatomical images in the same space (in both ways).

## First issues:​
- [ ]  ​Split the dataset and choose in which space to work (EPI or anatomical)
- [ ]  Build a generator to load the images and perform basic preprocs (intensity normalization, one hot encoding...).
- [ ]  Chose a build a network architecture (most likely some kind of U-Net).
